app/pages/api/puz/[[...puzzleId]].ts

```diff
@@ -3,11 +3,15 @@ import { exportFile } from '../../../lib/converter';
 import { getPuzzle } from '../../../lib/puzzleCache';
 
 export default async function puz(req: NextApiRequest, res: NextApiResponse) {
-  const { puzzleId } = req.query;
-  if (!puzzleId || Array.isArray(puzzleId)) {
+  const { urlTokens } = req.query;
+  const [ puzzleId ] = urlTokens || '';
+  console.log('here we are');
```

Comment:
Just a note to remove this extra logging

Reply:
Actually - I think maybe we should just leave this endpoint completely unchanged? Maybe this was an accidental commit. But this is what generates links to dl .puz files and I don't think we need to change those at all